WPCliCS/ruleset.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="WPCliCS" xsi:noNamespaceSchemaLocation="../vendor/squizlabs/php_codesniffer/phpcs.xsd">
+
+	<description>Coding standard for WP-CLI projects</description>
+
+	<!--
+	To include this ruleset in a WP CLI project, use rule ref="WPCliCS" in brackets.
```

Comment:
Syntax seems unclear here. Maybe something like this:
```suggestion
	To include this ruleset in a WP CLI project, use `<rule ref="WPCliCS">`.
```

Reply:
The problem is that you can't use `<` or `>` in XML comment without breaking it.